crates/ika-core/src/dwallet_mpc/mpc_outputs_verifier.rs

```diff
@@ -187,8 +144,15 @@ impl DWalletMPCOutputsVerifier {
         {
             session_output_data.current_result = OutputVerificationStatus::AlreadyCommitted;
             session_output_data.clear_data();
-            self.consensus_round_completed_sessions
-                .insert(session_info.session_identifier);
+            if let Err(e) = self
+                .consensus_round_completed_sessions_sender
+                .send(session_info.session_identifier)
+            {
+                error!(
+                    e=?e,
```

Comment:
```suggestion
                    error=?e,
```
Better be consistent with error=.. easier to look for it later in logs.